.openpublishing.redirection.json

```diff
@@ -1,5 +1,15 @@
 {
     "redirections": [
+        {
+            "source_path": "aspnetcore/tutorials/grpc/grpc-start.md",
```

Comment:
If this is the getting started tutorial then wouldn't grpc-start be a better URL?